analytics/secrets.tf

```diff
@@ -0,0 +1,25 @@
+module "kms-key" {
+  source = "../kms-key"
+
+  # Required
+  environment  = local.environment
+  project      = local.project
+  alias        = "analytics"
```

Comment:
alias have to be unique, perhaps make it project/env specific or make it a variables (with analytics as the default value)?

Reply:
yes! good point :)
the KMS module takes care of this for us
> name          = replace("alias/${var.project}/${var.environment}/${var.alias}", "/[^a-zA-Z0-9:///_-]+/", "-")